src/bundle/Resources/public/js/scripts/admin.location.bookmark.js

```diff
@@ -1,6 +1,30 @@
 (function (global, doc) {
     const SELECTOR_FORM = 'form[name="location_update_bookmark"]';
-    const updateBookmark = () => doc.querySelector(SELECTOR_FORM).submit();
+    const ID_UPDATE_BOOKMARK_CHECKBOX = 'location_update_bookmark_bookmarked';
```

Comment:
`const SELECTOR_BOOKMARK_CHECKBOX = '#location_update_bookmark_bookmarked';`

and please use `querySelector` instead of `getElementById`

Reply:
Why should I prefer `querySelector` over `getElementById`?

Reply:
1. For code consistency.
2. If you use `SELECTOR_BOOKMARK_CHECKBOX` and later someone will change the selector to let say `.my-new-selector` the code will still work (less to maintenance).